### PR TITLE
chore: Correct paths test, test-project pages

### PR DESCRIPTION
**Pull request by Josh-Walker-GM**
Corrects a minor error in #6924 where testing the pages did not use the correct fixture path for the test project case.

### CODE REVIEW

**Comment by Tobbe** [comment]
Dude! We can't keep up with all your PRs 😅 ❤️

File: packages/internal/src/__tests__/paths.test.ts [hunk 1]
```diff
@@ -890,78 +890,89 @@ describe('paths', () => {
...
-        expect(pages.length).toEqual(8)
-
-        const adminEditUserPage = pages.find(
-          (page) => page.importName === 'adminEditUserPage'
-        )
-        expect(adminEditUserPage).not.toBeUndefined()
-        expect(adminEditUserPage.importPath).toEqual(
-          importStatementPath(
-            path.join(pagesDir, 'admin/EditUserPage/EditUserPage')
+        console.log(pages)
```
Comment (Tobbe) [inline packages/internal/src/__tests__/paths.test.ts:897]: This can probably be removed, yeah?

**Comment by Josh-Walker-GM** [comment]
@Tobbe Sorry but at least this one fixes my own mistakes 😅

File: packages/internal/src/__tests__/paths.test.ts (hunk 1 above)
Comment (Josh-Walker-GM) [reply packages/internal/src/__tests__/paths.test.ts:897]: Yeah sorry

File: packages/internal/src/__tests__/paths.test.ts [hunk 2]
```diff
@@ -890,78 +890,87 @@ describe('paths', () => {
...
+          'HomePage',
+          'LoginPage',
+          'NotFoundPage',
+          'ProfilePage',
+          'ResetPasswordPage',
+          'SignupPage',
+          'WaterfallPage',
+        ]
+
+        for (let i = 0; i < pageNames.length; i++) {
```
Comment (Tobbe) [inline packages/internal/src/__tests__/paths.test.ts:914]: 100% nitpicking here, but we'd generally use `pageNames.forEach((pageName) => { ...`

**Comment by Josh-Walker-GM** [comment]
I only noticed this mistake from reading your discussion in #6919, the updated `paths.test.ts` should only rely on the global fixtures.

File: packages/internal/src/__tests__/paths.test.ts [hunk 3]
```diff
@@ -890,78 +890,87 @@ describe('paths', () => {
...
-          (page) => page.importName === 'FatalErrorPage'
-        )
-        expect(fatalErrorPage).not.toBeUndefined()
-        expect(fatalErrorPage.importPath).toEqual(
-          importStatementPath(
-            path.join(pagesDir, 'FatalErrorPage/FatalErrorPage')
-          )
-        )
+        for (let i = 0; i < scaffoldPageNames.length; i++) {
+          const pageName = scaffoldPageNames[i]
```
Comment (Tobbe) [inline packages/internal/src/__tests__/paths.test.ts:926]:
```suggestion
        scaffoldPageNames.forEach((pageName) => {
```

**Review by Tobbe — APPROVED** [review]

**Comment by Tobbe** [comment]
> I only noticed this mistake from reading your discussion in #6919, the updated `paths.test.ts` should only rely on the global fixtures.

So you're saying if I merge this I'll create even more work for myself (having to update #6919)
/me revokes approval 😛

File: packages/internal/src/__tests__/paths.test.ts (hunk 2 above)
Comment (Josh-Walker-GM) [reply packages/internal/src/__tests__/paths.test.ts]: More than happy to adopt the preferred choice

**Comment by Tobbe** [comment]
The CI failure we're seeing here seems to be happening on all PRs right now, so don't worry about it

**Pull request merged**